packages/nextjs/hooks/scaffold-stark/useDataTransaction.ts

```diff
@@ -0,0 +1,162 @@
+import { useMemo, useState, useEffect, useCallback } from "react";
+import { RpcProvider } from "starknet";
+import { useTargetNetwork } from "./useTargetNetwork";
+
+interface BlockData {
+  classeslength: number;
+  transaction: number;
+  blockStatus: string;
+  blockNumber: number;
+  blockHash: string;
+  blockVersion: string;
```

Comment:
I I think it is better to call it starknetVersion here, as it is really the version that is returned.